src/systems/detachable_joint/DetachableJoint.cc

```diff
@@ -249,10 +254,11 @@ void DetachableJoint::PreUpdate(
         // We do this by creating a detachable joint entity.
         this->detachableJointEntity = _ecm.CreateEntity();
 
+        std::string type = (this->weldChildToParent) ? "weld" : "fixed";
```

Comment:
Can you update the documentation in the `DetachableJoint` component that "weld" is now another option?